src/exits/zif_abapgit_exit.intf.abap

```diff
@@ -1,177 +1,156 @@
     IMPORTING
-      !iv_repo_url    TYPE csequence
-      !iv_repo_name   TYPE csequence
-      !iv_repo_key    TYPE csequence
+      !iv_repo_url TYPE csequence
+      !iv_repo_name TYPE csequence
+      !iv_repo_key TYPE csequence
       !iv_commit_hash TYPE zif_abapgit_git_definitions=>ty_sha1
     CHANGING
       !cv_display_url TYPE csequence
     RAISING
-      zcx_abapgit_exception.
-
+      zcx_abapgit_exception .
   METHODS adjust_display_filename
     IMPORTING
-      !is_repo_meta      TYPE zif_abapgit_persistence=>ty_repo
-      !iv_filename       TYPE string
+      !is_repo_meta TYPE zif_abapgit_persistence=>ty_repo
+      !iv_filename TYPE string
     RETURNING
-      VALUE(rv_filename) TYPE string.
-
+      VALUE(rv_filename) TYPE string .
   METHODS allow_sap_objects
     RETURNING
-      VALUE(rv_allowed) TYPE abap_bool.
-
+      VALUE(rv_allowed) TYPE abap_bool .
   METHODS change_local_host
     CHANGING
-      !ct_hosts TYPE zif_abapgit_definitions=>ty_string_tt.
-
+      !ct_hosts TYPE zif_abapgit_definitions=>ty_string_tt .
   METHODS change_max_parallel_processes
     IMPORTING
-      iv_package       TYPE devclass
+      !iv_package TYPE devclass
     CHANGING
-      cv_max_processes TYPE i.
-
+      !cv_max_processes TYPE i .
   METHODS change_proxy_authentication
     IMPORTING
-      !iv_repo_url             TYPE csequence
+      !iv_repo_url TYPE csequence
     CHANGING
-      !cv_proxy_authentication TYPE abap_bool.
-
+      !cv_proxy_authentication TYPE abap_bool .
   METHODS change_proxy_port
     IMPORTING
-      !iv_repo_url   TYPE csequence
+      !iv_repo_url TYPE csequence
     CHANGING
-      !cv_proxy_port TYPE string.
-
+      !cv_proxy_port TYPE string .
   METHODS change_proxy_url
     IMPORTING
-      !iv_repo_url  TYPE csequence
+      !iv_repo_url TYPE csequence
     CHANGING
-      !cv_proxy_url TYPE string.
-
+      !cv_proxy_url TYPE string .
   METHODS change_rfc_server_group
     CHANGING
-      cv_group TYPE rzlli_apcl.
-
+      !cv_group TYPE rzlli_apcl .
   METHODS change_supported_data_objects
     CHANGING
-      !ct_objects TYPE zif_abapgit_data_supporter=>ty_objects.
-
+      !ct_objects TYPE zif_abapgit_data_supporter=>ty_objects .
   METHODS change_supported_object_types
     CHANGING
-      !ct_types TYPE ty_object_types.
-
+      !ct_types TYPE ty_object_types .
   METHODS change_tadir
     IMPORTING
       !iv_package TYPE devclass
-      !ii_log     TYPE REF TO zif_abapgit_log
+      !ii_log TYPE REF TO zif_abapgit_log
+      !is_dot_abapgit TYPE zif_abapgit_dot_abapgit=>ty_dot_abapgit
+      !iv_ignore_subpackages TYPE abap_bool DEFAULT abap_false
+      !iv_only_local_objects TYPE abap_bool DEFAULT abap_false
     CHANGING
-      !ct_tadir   TYPE zif_abapgit_definitions=>ty_tadir_tt.
-
+      !ct_tadir TYPE zif_abapgit_definitions=>ty_tadir_tt .
   METHODS create_http_client
     IMPORTING
-      !iv_url          TYPE string
+      !iv_url TYPE string
```

Comment:
pretty print the file with indentation? will remove this change to the file

Reply:
indentation restored